interop/src/clients/corecrypto/ffi.rs

```diff
@@ -33,15 +32,18 @@ pub(crate) struct CoreCryptoFfiClient {
     prekey_last_id: u16,
 }
 
+const KEY_STORE_PATH: &str = "path";
+const KEY_STORE_KEY: &str = "path";
```

Comment:
This used to be `"key"` when it was in the `CoreCrypto::new` invocation as a literal.